src/_P118_Itho.ino

```diff
@@ -124,10 +124,10 @@ boolean Plugin_118(uint8_t function, struct EventStruct *event, String& string)
       break;
     }
 
-    case PLUGIN_SET_DEFAULTS: // Set defaults address to the one used in old versions of the library for backwards compatability
+    case PLUGIN_SET_DEFAULTS:  // Set defaults address to the one used in old versions of the library for backwards compatability
     {
-      PIN(0)     = -1;        // Interrupt pin undefined by default
-      PIN(1)     = 15;        // CS pin use the previous default of SS/gpio 15
+      PIN(0)     = -1;         // Interrupt pin undefined by default
+      PIN(1)     = PIN_SPI_SS; // CS pin use the previous default of PIN_SPI_SS/gpio 15
```

Comment:
The GPIO comment is only ESP8266

Reply:
I can add a conditional compile for ESP32 to set it to -1 by default. I would expect the default Slave Select to be the same for ESP32, when using the 'default' SPI pins. But I'd expect the user to select the desired CS pin anyway. Then 15 isn't a bad default, but also -1 wouldn't be wrong, just require them to pick one. Same situation for the interrupt pin.
The conditional definition for this value is ~stolen~borrowed from the Arduino source, so it works for ESP32 too.

Reply:
What I meant is that GPIO-15 is ESP8266 only, so a simple `(on ESP8266)` would be enough here in the comment line ;)
But indeed, had not thought of the default needed for ESP32.
If it is already a sensible value in the Arduino source then you don't need to do the #ifdef stuff.